frontend/src/components/UserSignUp/ReturnButton.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+export default function ReturnButton({ link }) {
+  const BackArrow = styled.div`
+    background-image: url("WhitePrevButton.png");
+    width: 40%;
+    height: 70%;
+    top: 0%;
+    right: 0%;
+    background-repeat: no-repeat;
+    background-position: center;
+  `;
```

Comment:
Can you move the styled components outside of this function (between lines 3 & 5)? The code compiles, but I see a bunch of warnings in the console when the StyedComponents are nested inside like this.